blocks/page_list/templates/itinerary/Itinerary.jsx

```diff
@@ -25,7 +29,7 @@ export default class Itinerary extends React.Component {
 
   constructor(...args) {
     super(...args);
-    this.state = props.itinerary || getItinerary();
+    this.state = this.props.itinerary || getItinerary();
```

Comment:
Both pre and post change are wrong. Props is the first argument to constructor -- it's the constructor, so nothing in `this` will have been set yet.

should be `constructor(props, ...args) { super(props, ...args); this.state = props.itinerary || getItinerary(); }`

https://facebook.github.io/react/docs/reusable-components.html#es6-classes